packages/fuel-gauge/src/predicate-conditional-inputs.test.ts

```diff
@@ -46,7 +46,7 @@ describe('PredicateConditionalInputs', () => {
     await tx1.waitForResult();
 
     // transfer base asset to Alice so she can pay the fees
-    const tx2 = await adminWallet.transfer(aliceWallet.address, 2_000, baseAssetId);
+    const tx2 = await adminWallet.transfer(aliceWallet.address, 2105, baseAssetId);
```

Comment:
Why did this increase?

Reply:
This was due to the `forc` upgrade. Often compiler changes can increase/decrease the size of the byte code due to the underlying changes. Here we saw an increase in the size of the bytecode, increasing the cost of the transaction, and `aliceWallet` then needed more funding to pay for the transaction.